octobot_trading/personal_data/portfolios/portfolio_manager.py

```diff
@@ -84,7 +85,7 @@ async def handle_balance_update_from_order(self, order, require_exchange_update:
         portfolio changes using order data (as in trading simulator)
         :return: True if the portfolio was updated
         """
-        if self.trader.is_enabled:
+        if self.trader.is_enabled and self._enable_portfolio_update_from_order:
```

Comment:
Could we add a test for it please? To be sure it won't be broken with future changes

Reply:
sure, i will

Reply:
done